example_app1.py

```diff
@@ -24,25 +21,30 @@
     ["Shoes", "Handbags", "Gloves", "Accessories"],
 )
 
-col1, col2, col3 = st.columns(3)
+col1, col2, col3, col4 = st.columns(4)
 
 measure: str = col1.radio("Measure", ["Sales", "Revenue [$]"])  # type: ignore
-compare_by = col2.radio("Campare by", ["Region", "Product"])
-coords = col3.radio("Coordinate system", ["Cartesian", "Polar"])
+compare_by = col2.radio("Compare by", ["Region", "Product", "Both"])
+coords = col3.radio("Coordinate system", ["Cartesian (desktop)", "Polar (mobile)"])
```

Comment:
What is the desktop/mobile for? Are they not supposed to work well on both platforms?

Reply:
Nevermind, I see you noted that in the PR message. Maybe, if that's the suggested use-case, you could say "best for desktop/best for mobile" to make it more clear what they mean.